bucket/foldingathome.json

```diff
@@ -0,0 +1,42 @@
+{
+    "version": "7.6.13",
+    "description": "Distributed computing project focused on disease research",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://foldingathome.org/support/faq/opensource"
+    },
+    "homepage": "https://foldingathome.org",
+    "url": "https://download.foldingathome.org/releases/public/release/fah-installer/windows-10-32bit/v7.6/fah-installer_7.6.13_x86.exe#/dl.7z",
+    "hash": "7618f1d98e1283442767f9735ae5f6c35a0c86b03c3ae62f45ee7be59509ec3e",
+    "pre_install": [
+        "Remove-Item \"$dir\\`$*\", \"$dir\\Uninstall.exe.nsis\" -Force -Recurse",
+        "if(!(Test-Path \"$persist_dir\\log.txt\")) { New-Item \"$dir\\log.txt\" | Out-Null }",
+        "if(!(Test-Path \"$persist_dir\\GPUs.txt\")) { New-Item \"$dir\\GPUs.txt\" | Out-Null }",
+        "if(!(Test-Path \"$persist_dir\\config.xml\")) { Set-Content \"$dir\\config.xml\" '<config></config>' | Out-Null }"
```

Comment:
Why it is so hard for you to be consistent??

https://github.com/lukesampson/scoop-extras/pull/4647#discussion_r477037653

Reply:
!

Reply:
https://github.com/lukesampson/scoop-extras/pull/4730#discussion_r485652896